### PR TITLE
adds ssl redirecting

### LINKED ISSUE #53: Add HTTPS redirecting
**Problem**
Currently if the dashboard is accessed with the HTTP protocol it will not redirect to HTTPS, this must happen for security reasons. 

**Requirements**
- [ ] Force https redirect if request is unsecure

### PR DESCRIPTION
**Pull request by RobertWSaunders**
fixes #53  

enviroment must be in production and FORCE_SSL must be set

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
@zackharley this will add redirecting to HTTPS. 

After talking with @ccrowley96 the `secure: always` is not a viable option for us as the Google App Engine Node.js runtime does not support it.

File: server/index.js [hunk 1]
```diff
@@ -46,9 +47,24 @@ db((err, db) => {
 	app.use(webhook());
 	app.use(mailer());
 
+	// Authentication
+	app.use(auth());
```
Comment (zackharley) [inline server/index.js]: Why was auth moved to here? Shouldn't it only be applied to the API?

Comment (RobertWSaunders) [reply server/index.js]: Will fix this, was messing around and forgot to switch back. Good catch 👍

**Comment by zackharley** [comment]
@RobertWSaunders Looks good. Will merge when tests pass!

**Comment by zackharley** [comment]
screen shot 2017-11-25 at 2 31 58 pm

**Comment by zackharley** [comment]
@RobertWSaunders There is a conflict with `package-lock.json`

**Pull request merged**